portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentStagingUpgrade.testcase

```diff
@@ -15,7 +15,7 @@ definition {
 		SearchAdministration.executeReindex();
 	}
 
-	@priority = 5
+	@priority = 3
```

Comment:
I think upgrade tests should be FP4 at least.

Reply:
Set to FP4